mobile/lib/common/routes.dart

```diff
@@ -26,6 +26,33 @@ GoRouter createRoutes() {
       navigatorKey: rootNavigatorKey,
       initialLocation: LoadingScreen.route,
       routes: <RouteBase>[
+        GoRoute(
```

Comment:
Given the size of this PR you could have spared us some of the changes in this PR (e.g. moving things around). 

Potentially, you could even split it up into two PRs. First dealing with the backup and then with the restore.

Reply:
No worries I can split up the cosmetic changes.

If you really think that it will improve the review, I can also split up the backup and restore changes, but that will be more complicated.

Reply:
I think it would have been easier. Now I'm almost through 😅